### PR TITLE
Prepend all PyTorch QNNPACK symbols with pytorch_ to avoid symbol collision.

### PR DESCRIPTION
**Pull request by AshkanAliabadi**
This probably doesn't catch absolutely everything but it's a good starting point.

### CODE REVIEW

**Comment by supriyar** [comment]
Did we test this using unit tests? 
Can we also run the python tests with your changes from https://github.com/pytorch/pytorch/pull/25844 stacked on this to make sure we're using the correct symbols in pytorch as well.

File: aten/src/ATen/native/quantized/cpu/qnnpack/src/qnnpack/q8gemm.h [hunk 1]
```diff
@@ -30,20 +30,20 @@ extern "C" {
       size_t c_stride,                           \
       const union pytorch_qnnp_conv_quantization_params* quantization_params);
 
-DECLARE_Q8GEMM_UKERNEL_FUNCTION(q8gemm_ukernel_3x3c8__neon)
-DECLARE_Q8GEMM_UKERNEL_FUNCTION(q8gemm_ukernel_2x4c8__neon)
-DECLARE_Q8GEMM_UKERNEL_FUNCTION(q8gemm_ukernel_4x8__neon)
-DECLARE_Q8GEMM_UKERNEL_FUNCTION(q8gemm_ukernel_6x4__neon)
-DECLARE_Q8GEMM_UKERNEL_FUNCTION(q8gemm_ukernel_8x8__neon)
+PYTORCH_DECLARE_Q8GEMM_UKERNEL_FUNCTION(pytorch_q8gemm_ukernel_3x3c8__neon)
```
Comment (ljk53) [inline aten/src/ATen/native/quantized/cpu/qnnpack/src/qnnpack/q8gemm.h]: I might call it DELCARE_PYTORCH_XXX but lets land first to unblock other work.

**Review by ljk53 — APPROVED** [review]
Looks like these are simply adding prefix to symbols under aten/src/ATen/native/quantized/cpu/qnnpack. Stamp it as it won't affect things outside this directory.

**Comment by dzhulgakov** [comment]
Looks good. For testing - you probably can run objdump/nm on the resulting static library and see whether anything is left

**Review by supriyar — APPROVED** [review]

**Comment by AshkanAliabadi** [comment]
Merged into https://github.com/pytorch/pytorch/pull/25844.

**Pull request closed without merging**